src/routes/chat/components/message/Message.js

```diff
@@ -0,0 +1,74 @@
+import React, { memo } from 'react';
+import useStyles from 'isomorphic-style-loader/useStyles';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+// After creating user model will use Avatar component instead <img>
+// import Avatar from '../avatar';
+import IconMessageStatus from '../messageStatus/IconMessageStatus';
+
+import s from './Message.css';
+
+const testIsRead = true;
+
+function Message({ isMe, createdAt, text, user }) {
+  useStyles(s);
+  return (
+    <div
+      className={classNames('message', {
+        [s.isMe]: isMe,
+      })}
+    >
+      <div className={s.message}>
+        <span
+          className={classNames('', {
```

Comment:
Static empty string is redundant here

Reply:
Change to "className={classNames( {"   or "className={classNames(null, {" ?

Reply:
First option, please. `classNames` ignores any falsy values.